docs/database/usage/index.md

```diff
@@ -335,3 +335,38 @@ and caching. To disable this functionality, update the `database_persistence_ena
 ```
 
 To enable persistence, view the [Offline Support](/database/offline-support) documentation.
+
+# Change the database URL
+
+## Switch the database URL of the default app
```

Comment:
This can be removed with the header changes.

Reply:
done